test/settings_test.dart

```diff
@@ -93,6 +93,8 @@ void main() {
       when(mockAuthProvider.getProfilePictureURL())
           .thenAnswer((_) => Future.value(null));
       when(mockAuthProvider.isVerified).thenAnswer((_) => Future.value(true));
+      when(mockAuthProvider.currentUserFromCache).thenReturn(User(
```

Comment:
Please add a test somewhere in this file to test that the Admin Panel button shows if the user has enough permissions, but doesn't show if they don't.

You could also add a widget test for RequestCard (remember we talked about it - basically test that if you press approve, it shows as approved, if you press deny it shows as denied etc) to learn how to do widget testing:
https://flutter.dev/docs/cookbook/testing/widget/introduction
This isn't mandatory though, but it's recommended. Up to you.